### PR TITLE
Remove nectar's 60 second timeout until a transaction is mined

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

File: nectar/src/ethereum/wallet.rs [hunk 1]
```diff
@@ -346,6 +306,12 @@ impl Wallet {
         self.geth_client.get_balance(self.account()).await
     }
 
+    pub(crate) async fn erc20_balance(&self, token_contract: Address) -> anyhow::Result<Erc20> {
```
Comment (D4nte) [inline nectar/src/ethereum/wallet.rs:309]: I believe this is useless in the context of a binary crate.

File: nectar/src/ethereum/wallet.rs [hunk 2]
```diff
@@ -355,6 +321,36 @@ impl Wallet {
...
+
+    async fn wait_until_transaction_receipt(
+        &self,
+        transaction_hash: Hash,
+        chain: ChainId,
+    ) -> anyhow::Result<TransactionReceipt> {
+        let poll_interval = match chain {
+            ChainId::MAINNET => 10, // roughly half the blocktime
+            ChainId::KOVAN => 2,    // roughly half the blocktime
+            ChainId::ROPSTEN => 10, // kovan has inconsistent blocktime between 1 sec and 1 min
```
Comment (D4nte) [inline nectar/src/ethereum/wallet.rs:338]:
```suggestion
            ChainId::ROPSTEN => 10, // ropsten has inconsistent blocktime between 1 sec and 1 min
```

Or did you actually mean Kovan?

**Review by D4nte — APPROVED** [review]

File: nectar/src/ethereum/wallet.rs (hunk 2 above)
Comment (thomaseizinger) [reply nectar/src/ethereum/wallet.rs:338]: I did not :)

File: nectar/src/ethereum/wallet.rs (hunk 1 above)
Comment (thomaseizinger) [reply nectar/src/ethereum/wallet.rs:309]: True, is an artifact of amending several commits :sweat_smile:

**Review by da-kami — APPROVED** [review]

**Comment by thomaseizinger** [comment]
bors retry

**Pull request merged**